### DISCUSSION TITLE
Can this XplatGenerateReleaseNotes extension be used on a Linux build agent through ADO

### DISCUSSION THREAD

**Discussion by deepakatlX** [rfennell/AzurePipelines#1457] [Q&A]
Will this XplatGenerateReleaseNotes extension work on a Linux build agent through in Azure Pipeline. I'm using a Linux build agent and want to use this extension. I went through the documentation and didn't find anything regarding agents except for being cross platform.   Can anyone confirm please if this task can be run on Linux agent for ADO?

**Comment by rfennell** [comment]
The XplatGenerateReleaseNotes task is tested on Windows, Linux and MacOS Agents. 

It replaced the GenerateReleaseNotes that was Windows only
